src/apifairy/core.py

```diff
@@ -147,14 +146,10 @@ def resolver(schema):
         ma_plugin.converter.field_mapping[fields.URLFor] = ('string', 'url')
         ma_plugin.converter.field_mapping[fields.AbsoluteURLFor] = \
             ('string', 'url')
-        if sqla is not None:  # pragma: no cover
+        if sqla is not None:
             ma_plugin.converter.field_mapping[sqla.HyperlinkRelated] = \
                 ('string', 'url')
 
-        # configure FileField
-        ma_plugin.converter.field_mapping[apifairy_fields.FileField] = \
-            ('string', 'binary')
```

Comment:
I think you have some merging issues in this PR, because I don't see why you would delete this. My guess is that you are working off of an older version of this code that doesn't have this change.